masonite/snippets/auth/controllers/RegisterController.py

```diff
@@ -5,7 +5,7 @@
 from masonite.helpers import password as bcrypt_password
 from masonite.request import Request
 from masonite.view import View
-
+from masonite.auth.MustVerifyEmail import MustVerifyEmail
```

Comment:
add this authentication class to the __init__.py file so we can just do `from masonite.auth import MustVerifyEmail` instead of this longer more redundant form.